packages/drafts-realm/garden-design.gts

```diff
@@ -218,27 +184,70 @@ class Isolated extends Component<typeof GardenDesign> {
 
   defaultCount = 10;
   defaultUnitSize = 50;
-  optionsMap = new Map(OPTIONS.map((el) => [el.id, el]));
 
   @tracked cols = this.args.model.columns ?? this.defaultCount;
   @tracked rows = this.args.model.rows ?? this.defaultCount;
   @tracked gridStyle = htmlSafe(`
     grid-template-columns: repeat(${this.cols}, ${this.defaultUnitSize}px);
     grid-template-rows: repeat(${this.rows}, ${this.defaultUnitSize}px);
   `);
-  @tracked gridMap: TrackedMap<string, Option | null>;
+  @tracked gridMap: TrackedMap<string, GardenItem | null>;
+
+  @tracked
+  private declare liveQuery: {
+    instances: CardDef[];
+    isLoading: boolean;
+  };
 
   constructor(owner: Owner, args: any) {
     super(owner, args);
     this.gridMap = new TrackedMap(
       this.generateGridIds().map((id) => {
         let maybeOption = this._gridModel.find(
-          (el: [string, Option | null]) => el[0] === id,
+          (el) => this.composeId(el.position.x, el.position.y) === id,
         );
-        return [id, maybeOption ? maybeOption[1] : null];
+        return [id, maybeOption ? (maybeOption.card as GardenItem) : null];
       }),
     );
     this.updateGridModel();
+    let type = {
+      module: 'http://localhost:4201/drafts/garden-design',
+      name: 'GardenItem',
```

Comment:
I need a query using a type filter, but I don't know how to make the module URL work in all environments. I believe this module URL works only locally. Does anyone know how to make the module URL environment-agnostic?

Reply:
Would something like this help https://github.com/cardstack/boxel/pull/1368/files#diff-32330c02279a76cb3e85d743f9cc98dd41813d5949eef273e7cd50dcc0b838a2R41-R59?

Reply:
Nvm, I can just use `_cardType` for this. [412140e](https://github.com/cardstack/boxel/pull/1370/commits/412140e0bc3c5791f9d9f3259b5aa720c80ca05c)